raiden/raiden_service.py

```diff
@@ -411,16 +412,7 @@ def install_and_query_payment_network_filters(
                 token_network_registry,
                 from_block,
             )
-
-            chain_id = self.chain.network_id
-            for event in self.blockchain_events.poll_blockchain_events(
-                self.get_block_number(),
-            ):
-                on_blockchain_event(
-                    self,
-                    event, event.event_data['block_number'],
-                    chain_id,
-                )
+            # the events will be fetched as the alarm task fetches the missed blocks
```

Comment:
The alarm task doesnt really learn about missed blocks, just about the new blocks, this is a bit better IMO: 

> the events will be fetched on the first call to `poll_blockchain_events`, done once the alarm task learns about a new block

Reply:
Hmm, right, I'll change the comment, but the filter will still get the events in that range, in the first call with a up-to-date `current_block_number`.